### ISSUE TITLE
(Trivial) How to change the colour of the canvas background?

### ISSUE THREAD

**Issue by hyanwong** [kitchensjn/tskit_arg_visualizer#26]
When plotting on a page that has a dark background, it would be nice to be able to set the default canvas colour (so the background becomes e.g. white). Here's what a test case looks like on the tskit tutorials website (based on jupyterbook), in which we tend to draw plots with white background.

<img width="691" alt="Screenshot 2023-08-01 at 12 36 40" src="https://github.com/kitchensjn/tskit_arg_visualizer/assets/4699014/4283f039-52f2-4297-8cea-a57b9eb0dab9">

There's presumably an easy css spec I can use to set the d3js background colour?

**Comment by kitchensjn** [comment]
We can either set the SVG background-color to white or the entire div, which contains the buttons as well. I've gone with the latter as there may be scenarios where having an SVG with a transparent background could still be nice.

**Comment by hyanwong** [comment]
Great, thanks.

Slightly related: I guess it should be easy to hide the buttons using css? Maybe worth noting how to do that in the docs?

**Comment by kitchensjn** [comment]
I'm hesitant to resize the SVG based on the window size as that could make reproducible figures more difficult, but I'd be up for discussing this more. I've just updated visualizer to give it better formatting within notebooks. When the width of the visualizer is smaller than the width of the output div, then the visualizer is centered within the output div using flexbox. Alternatively, it is now scrollable when the width of the visualizer exceeds the width of the output div.

I also added another button to hide/show the visualizer buttons to reduce some of the clutter, especially if we add more controls. I'll eventually come back and improve the styling of this button dashboard.
